src/sentry/tasks/post_process.py

```diff
@@ -171,10 +173,16 @@ def _capture_group_stats(job: PostProcessJob) -> None:
         metrics.incr("events.unique", tags=tags, skip_internal=False)
 
 
-def should_issue_owners_ratelimit(project_id, group_id):
+def should_issue_owners_ratelimit(project_id: int, group_id: int, organization_id: int | None):
     """
     Make sure that we do not accept more groups than ISSUE_OWNERS_PER_PROJECT_PER_MIN_RATELIMIT at the project level.
     """
+    enforced_limit = ISSUE_OWNERS_PER_PROJECT_PER_MIN_RATELIMIT
+    if organization_id and organization_id in configuration.get(
+        "org_ids_with_increased_issue_owners_ratelimit", set()
+    ):
+        enforced_limit = HIGHER_ISSUE_OWNERS_PER_PROJECT_PER_MIN_RATELIMIT
+
```

Comment:
Wondering if this should just be a feature flag instead. We'd still back it with a list of orgs in getsentry, but then the interface is simpler

Reply:
I had thought that using flagr in post-process was unreliable? Feature flagging would make more sense if not.

Reply:
We can't use flagr, but you can define your own feature flag handler in getsentry that checks the list. Potentially you could even just add them to `HardcodedEarlyAdopterHandler`

Reply:
TIL, thanks I'll refactor this